src/dashboards/videos/course.jsonnet

```diff
@@ -0,0 +1,315 @@
+      the video during the first seconds of the video. This time range is
+      controlled by the `View count threshold` variable.
+    |||,
+    datasource=common.constants.lrs,
+    graphMode='none',
+    reducerFunction='sum'
+  ).addTarget(
+    elasticsearch.target(
+      datasource=common.constants.lrs,
+      query='%(course_query)s AND verb.id:"%(verb_played)s" AND %(time)s:[0 TO $VIEW_COUNT_THRESHOLD]' % {
+        course_query: common.queries.school_course_session,
+        verb_played: common.constants.verb_id_played,
+        time: common.utils.single_escape_string(common.fields.result_extensions_time),
+      },
+      metrics=[common.objects.count_metric],
+      bucketAggs=[common.objects.date_histogram('$STATEMENTS_INTERVAL')],
+      timeField='timestamp'
+    )
+  ),
+  gridPos={ x: 0, y: 9, w: 6, h: 9 }
+)
+.addPanel(
+  statPanel.new(
+    title='Complete views',
+    description=|||
+      Total number of complete views of videos present in the selected course / session.
+      A view is considered as complete when the completion threshold of the video has been reached.
+    |||,
+    datasource=common.constants.lrs,
+    graphMode='none',
+    reducerFunction='sum',
+  ).addTarget(
+    elasticsearch.target(
+      datasource=common.constants.lrs,
+      query='%(course_query)s AND verb.id:"%(verb_completed)s"' % {
+        course_query: common.queries.school_course_session,
+        verb_completed: common.constants.verb_id_completed,
+      },
+      metrics=[common.objects.count_metric],
+      bucketAggs=[common.objects.date_histogram()],
+      timeField='timestamp'
+    )
+  ),
+  gridPos={ x: 6, y: 9, w: 6, h: 9 }
+)
+.addPanel(
+  graphPanel.new(
+    title='Views by ${STATEMENTS_INTERVAL}',
+    description=|||
+      A view is counted when the user has clicked the play button in the interface
+      in the first ${VIEW_COUNT_THRESHOLD} seconds of the video.
+    |||,
+    datasource=common.constants.lrs,
+  ).addTarget(
+    elasticsearch.target(
+      datasource=common.constants.lrs,
+      query='%(course_query)s AND verb.id:"%(verb_played)s" AND %(time)s:[0 TO $VIEW_COUNT_THRESHOLD]' % {
+        course_query: common.queries.school_course_session,
+        verb_played: common.constants.verb_id_played,
+        time: common.utils.single_escape_string(common.fields.result_extensions_time),
+      },
+      metrics=[common.objects.count_metric],
+      bucketAggs=[common.objects.date_histogram('$STATEMENTS_INTERVAL')],
+      timeField='timestamp'
+    )
+  ),
+  gridPos={ x: 12, y: 9, w: 12, h: 9 }
+)
+.addPanel(
+  {
+    title: 'Statements by user',
+    description: |||
+      The count of statements by user.
+      On the X-axis is the number of statements.
+      On the Y-axis is the number of users.
+    |||,
+    datasource: common.constants.lrs,
+    fieldConfig: {
+      defaults: {
+        color: {
+          mode: 'palette-classic',
+        },
+        displayName: 'User',
+        mappings: [],
```

Comment:
```suggestion
```

Reply:
Nice catch) Yes, indeed this can be removed) Thanks)